virtual/wine/metadata.xml

```diff
@@ -6,6 +6,7 @@
 		<name>Wine</name>
 	</maintainer>
 	<use>
+		<flag name="proton">Enable Valve Software's fork of Wine from Proton</flag>
```

Comment:
nitpick but "from proton" sounds a little weird to me, maybe

Enable Proton, Valve Software's fork of Wine

or something

Reply:
Was trying to avoid making it sound like is /is/ proton (we're not enabling https://github.com/ValveSoftware/Proton, that's not packaged), albeit non-native English speaker so unsure for best wording.

Arch even call this package wine-valve to avoid the confusion, but upstream calls it proton-wine so well and it's recognizable so I preferred that.

Reply:
Maybe

 Based on Valve Software's fork of Wine, Proton

 or something like that?

Reply:
Maybe there's no need to mention proton at all, it's already the USE name.

aka just "Enable Valve Software's fork of Wine"

Reply:
sure